src/System.Windows.Forms.Primitives/src/System/Windows/Forms/HighDpiMode.cs

```diff
@@ -37,8 +37,9 @@ public enum HighDpiMode
         SystemAware,
 
         /// <summary>
-        ///  The window checks for DPI when it's created and adjusts scale factor when the DPI changes.
+        /// The window checks for DPI when it's created and adjusts scale factor when the DPI changes.
         /// </summary>
+        [Obsolete("Permonitor mode is applicable only on Windows 8.1. Winforms does not support this mode", true)]
```

Comment:
```suggestion
        [Obsolete("PerMonitor mode is applicable only on Windows 8.1", true)]
```

1. What would be the experience if an app is build/run on Windows 8.1?
2. The obsoletion must contain diagnostics as described in https://github.com/dotnet/designs/blob/main/accepted/2020/better-obsoletion/better-obsoletion.md. https://github.com/dotnet/runtime/blob/main/docs/project/list-of-diagnostics.md may be useful as well.